packages/git-semver-tags/index.js

```diff
@@ -38,6 +41,12 @@ module.exports = function(callback, opts) {
           if (lernaTag(tag, opts.package)) {
             tags.push(tag);
           }
+        } else if (opts.tagPrefix) {
+          if (tagPrefixRegexp.test(tag)) {
+            if (semverValid(tag.replace(tagPrefixRegexp, ''))) {
```

Comment:
Could you use `String.match()` with a modified regex to reduce these two conditions down to one?

For example:

```javascript
const matches = tag.match(new RegExp(`^${opts.tagPrefix}(.*)`));
if (matches && semverValid(matches[0])) {}
```